source/future.tex

```diff
@@ -51,6 +51,15 @@
 \rSec1[depr.c.headers]{C standard library headers}
 
 \pnum
+For compatibility with the C standard library\indextext{library!C standard},
+the \Cpp standard library provides headers
+\tcode{<ccomplex>}~(\ref{depr.ccomplex.syn}),
```

Comment:
We're still in a section "C standard library headers" here, yet we keep talking about C++ headers.
Either rename this section to "Standard library headers" or add a section "C++ standard library headers" that contains the C++-only content. (I'm unsure what remains in the existing "C headers" section in the latter case.)